PROJECT_LIFECYCLE.md

```diff
@@ -0,0 +1,121 @@
+# Project Lifecycle
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Projects](#projects)
+- [Project Lifecycle Process](#project-lifecycle-process)
+- [Project Lifecycle Stages](#project-lifecycle-stages)
+- [References](#references)
+
+## Overview
+
+Eiffel Community is home to various projects that enable the members of the community to use, contribute,
+and collaborate on Eiffel.
+
+This document describes how Eiffel Community defines the projects, how the projects can be created, maintained,
+and archived. In addition to the project lifecyle process, stages which the projects can be admitted or transitioned
+into as well as criteria and expectations for the stages are described.
+
+## Projects
+
+The activities of the Eiffel Community are organized into projects targeting different areas within the scope of
+Eiffel Community including but not limited to development and maintenance of Eiffel Protocol, development of new
+functionality that adheres to the protocol, integration of other technologies, testing, documentation, as well as
+the support functions to create and maintain the infrastructure and the toolchain community uses.
+
```

Comment:
Refer to PROJECTS.md here, where the current list of projects can be seen?

Reply:
done